ci-operator/step-registry/baremetalds/e2e/upgrade/baremetalds-e2e-upgrade-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: baremetalds-e2e-upgrade
+  steps:
+    allow_skip_on_success: true 
+    pre:
+    - ref: baremetalds-packet-setup
+    - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-setup
+    test:
+      - ref: baremetalds-e2e-conf
+      - ref: baremetalds-e2e-test
+      - ref: baremetalds-e2e-upgrade-upgrade
+      # - ref: baremetalds-e2e-test
+    post:
+    - ref: baremetalds-devscripts-gather
+    - ref: baremetalds-packet-teardown
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"   
+    env:
+      DEVSCRIPTS_CONFIG: |
+        OPENSHIFT_RELEASE_IMAGE=$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
```

Comment:
I don't think this kind of indirection is supported. In any case it should not be necessary to set DEVSCRIPTS_CONFIG but I think it would be sufficient something like `dependencies: RELEASE_IMAGE_LATEST: "release:initial"` (see https://github.com/openshift/release/blob/8cb0233ac6d7a0a4c3b445fae410610ecd7d72b3/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L94)